staramr/databases/resistance/pointfinder/complex/ComplexMutations.py

```diff
@@ -54,7 +54,8 @@ def get_matches(self, results_table, hit):
                 result = [hit.get_genome_id(),
                         ", ".join(intersection), # Technically these are also Pointfinder co-ords
                         row.phenotype,
-                        "complex",
+                        "None",  # CGE-predicted phenotype
```

Comment:
Sorry. By changing to `None` I meant the keyword in Python `None` and not the string `"None"`.

The issue I'm trying to resolve is that the staramr code currently has multiple ways of representing missing data (as the character `-`, as pandas `pd.NA`, as an empty string, etc). Adding the string `"None"` or the string `"N/A"` would be adding yet another way to represent missing data.

Ideally, I'd like all data in pandas dataframes to represent missing data using pandas `pd.NA` (or `NaN` for numbers).

That way, the specific character to use for missing data can all be controlled when exporting the pandas dataframe to a file using the `na_rep` parameter. For example:

https://github.com/phac-nml/staramr/blob/e9b200cae9ea67ce8edebe54500d836c20f05dfa/staramr/subcommand/Search.py#L241

Here, we set `na_rep=self.BLANK` which corresponds to `-`.

Since there's a number of places we are setting the character for missing data in staramr currently, I don't want to try and remove them all in this PR. But, I would like to try and avoid adding yet another missing data representation to our output, by using the Python keyword `None` (which I believe gets converted to `pd.NA` when loaded into a pandas dataframe).

Reply:
Gotcha. Updated.